.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Team Lead
+* @hoangmirs
+
+# Team Members
+* @andyduong1920 @byhbt @hanam1ni @junan @longnd @rosle @Nihisil @bterone @Lahphim @malparty
```

Comment:
Because of [this issue](https://github.com/nimblehq/gin-templates/pull/75#pullrequestreview-930100069),  we should put these 2 sections into one.
And how did you define the team members for this project? Do they all know about AWS and Terraform? 🤔 
Or should we add the web-chaper group instead?

Reply:
I didn't know that. Fixed in 7315c3b. Thank you.

Reply:
> Do they all know about AWS and Terraform?

yes, I made the list based on that. In fact, Alex doesn't know Terraform but he is learning it.